crates/chia-consensus/src/gen/validation_error.rs

```diff
@@ -5,47 +5,156 @@ use thiserror::Error;
-    AssertBeforeSecondsRelative,
-    AssertBeforeHeightAbsolute,
-    AssertBeforeHeightRelative,
-    AssertSecondsAbsolute,
-    AssertSecondsRelative,
-    AssertMyAmountFailed,
-    AssertMyBirthSecondsFailed,
-    AssertMyBirthHeightFailed,
-    AssertMyPuzzlehashFailed,
-    AssertMyParentIdFailed,
     AssertMyCoinIdFailed,
     AssertPuzzleAnnouncementFailed,
     AssertCoinAnnouncementFailed,
+    AssertHeightRelativeFailed,
+    AssertHeightAbsoluteFailed,
+    AssertSecondsAbsoluteFailed,
+    CoinAmountExceedsMaximum,
+    SexpError,
+    InvalidFeeLowFee,
+    MempoolConflict,
+    MintingCoin,
+    ExtendsUnknownBlock,
+    CoinbaseNotYetSpendable,
+    /// Renamed from "BlockCostExceedsMax" since it's more generic than that.
+    CostExceeded,
+    BadAdditionRoot,
+    BadRemovalRoot,
+    InvalidPospaceHash,
+    InvalidCoinbaseSignature,
+    InvalidPlotSignature,
+    TimestampTooFarInPast,
+    TimestampTooFarInFuture,
+    InvalidTransactionsFilterHash,
+    InvalidPospaceChallenge,
+    InvalidPospace,
+    InvalidHeight,
+    InvalidCoinbaseAmount,
+    InvalidMerkleRoot,
+    InvalidBlockFeeAmount,
+    InvalidWeight,
+    InvalidTotalIters,
+    BlockIsNotFinished,
+    InvalidNumIterations,
+    InvalidPot,
+    InvalidPotChallenge,
+    InvalidTransactionsGeneratorHash,
+    InvalidPoolTarget,
+    InvalidCoinbaseParent,
+    InvalidFeesCoinParent,
+    ReserveFeeConditionFailed,
+    NotBlockButHasData,
+    IsTransactionBlockButNoData,
+    InvalidPrevBlockHash,
+    InvalidTransactionsInfoHash,
+    InvalidFoliageBlockHash,
+    InvalidRewardCoins,
+    InvalidBlockCost,
+    NoEndOfSlotInfo,
+    InvalidPrevChallengeSlotHash,
+    InvalidSubEpochSummaryHash,
+    NoSubEpochSummaryHash,
+    ShouldNotMakeChallengeBlock,
+    ShouldMakeChallengeBlock,
+    InvalidChallengeChainData,
+    InvalidCcEosVdf,
+    InvalidRcEosVdf,
+    InvalidChallengeSlotHashRc,
+    InvalidPriorPointRc,
+    InvalidDeficit,
+    InvalidSubEpochSummary,
+    InvalidPrevSubEpochSummaryHash,
+    InvalidRewardChainHash,
+    InvalidSubEpochOverflow,
+    InvalidNewDifficulty,
+    InvalidNewSubSlotIters,
+    InvalidCcSpVdf,
+    InvalidRcSpVdf,
+    InvalidCcSignature,
+    InvalidRcSignature,
+    CannotMakeCcBlock,
+    InvalidRcSpPrevIp,
+    InvalidRcIpPrevIp,
+    InvalidIsTransactionBlock,
+    InvalidUrsbHash,
+    OldPoolTarget,
+    InvalidPoolSignature,
+    InvalidFoliageBlockPresence,
+    InvalidCcIpVdf,
+    InvalidRcIpVdf,
+    IpShouldBeNone,
+    InvalidRewardBlockHash,
+    InvalidMadeNonOverflowInfusions,
+    NoOverflowsInFirstSubSlotNewEpoch,
+    MempoolNotInitialized,
+    ShouldNotHaveIcc,
+    ShouldHaveIcc,
+    InvalidIccVdf,
+    InvalidIccHashCc,
+    InvalidIccHashRc,
+    InvalidIccEosVdf,
+    InvalidSpIndex,
+    TooManyBlocks,
+    InvalidCcChallenge,
+    InvalidPrefarm,
+    AssertSecondsRelativeFailed,
+    BadCoinbaseSignature,
+    // InitialTransactionFreeze (removed)
```

Comment:
if you want to maintain the same enum values, you still need a place holder here

Reply:
This is commented in chia-blockchain as well.